src/main/java/com/internet/shop/dao/jdbc/DriverDaoJdbcImpl.java

```diff
@@ -0,0 +1,114 @@
+        String getQuery = "SELECT * FROM drivers WHERE driver_id=? AND deleted = FALSE";
+        Driver driver = null;
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getQuery)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                driver = createDriver(resultSet);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get driver id + " + id + " from DB", e);
+        }
+        return Optional.ofNullable(driver);
+    }
+
+    @Override
+    public Driver update(Driver driver) {
+        String updateQuery = "UPDATE drivers SET driver_name = ?, license_number = ? "
+                + "WHERE driver_id = ? AND deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(updateQuery)) {
+            statement.setString(1, driver.getName());
+            statement.setString(2, driver.getLicenceNumber());
+            statement.setLong(3, driver.getId());
+            statement.executeUpdate();
+            return driver;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't update " + driver + " in DB", e);
+        }
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String deleteQuery = "UPDATE drivers SET deleted = ?  WHERE driver_id = ?";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(deleteQuery)) {
+            statement.setBoolean(1, true);
+            statement.setLong(2, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't delete driver id "
+                    + id + " from DB", e);
+        }
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String getAllQuery = "SELECT * FROM drivers WHERE deleted = FALSE";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(getAllQuery)) {
+            ResultSet resultSet = statement.executeQuery();
+            List<Driver> allDrivers = new ArrayList<>();
+            while (resultSet.next()) {
+                allDrivers.add(createDriver(resultSet));
+            }
+            return allDrivers;
+        } catch (SQLException e) {
+            throw new RuntimeException("Can`t get all drivers from DB ", e);
+        }
+    }
+
+    private Driver createDriver(ResultSet resultSet) {
+        try {
+            String name = resultSet.getString("driver_name");
+            String license = resultSet.getString("license_number");
+            Long id = resultSet.getObject("driver_id", long.class);
```

Comment:
```suggestion
            Long id = resultSet.getObject("driver_id", Long.class);
```
use wrapper class everywhere